dropwizard-app/src/main/java/org/coner/core/ConerCoreService.java

```diff
@@ -10,32 +10,60 @@
 import java.util.List;
 
 /**
- *
+ * The ConerCoreService is the domain level implementation of all functionality in the Coner core project.
```

Comment:
As in this one particular javadoc line?  I think it's pretty concise and understandable.